pkg/kubelet/kubelet.go

```diff
@@ -2465,7 +2465,7 @@ func (kl *Kubelet) syncLoopIteration(ctx context.Context, configCh <-chan kubety
 		ready := update.Result == proberesults.Success
 		kl.statusManager.SetContainerReadiness(update.PodUID, update.ContainerID, ready)
 
-		status := ""
+		status := "unready"
```

Comment:
It seems reasonable from a logical standpoint. However, I've noticed something. Are we using "unready" or "unhealthy" as the default value? The startup prober default value is "unhealthy." I find this quite interesting. I haven't closely examined the code, but should we stick with "unhealthy" or switch to something like "unready" or "unstart" for better semantic clarity?



```go
case update := <-kl.readinessManager.Updates():
	ready := update.Result == proberesults.Success
	kl.statusManager.SetContainerReadiness(update.PodUID, update.ContainerID, ready)

	status := "unready"
	if ready {
		status = "ready"
	}
	handleProbeSync(kl, update, handler, "readiness", status)
case update := <-kl.startupManager.Updates():
	started := update.Result == proberesults.Success
	kl.statusManager.SetContainerStartup(update.PodUID, update.ContainerID, started)

	status := "unhealthy"
	if started {
		status = "started"
	}
	handleProbeSync(kl, update, handler, "startup", status)
```

Reply:
@ffromani @bart0sh any suggestions? :)

Reply:
`unhealthy` sounds reasonable and consistent to me.

Reply:
Thanks for your suggestions, i will change to `unhealthy`.